Bottom: 883bbe03c4b6b57b2ff24024ecaf831cee468f05
Top:    c02a3fe189d6ee943a063df10643a3de3cb3981c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-04 23:42:19 +0000

Refresh of draw-pin-and-via-hole-cores-in

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 6b6d318..3294a24 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1638,7 +1638,6 @@ via_hole_cyl_callback (const BoxType * b, void *cl)
 void
 ghid_draw_everything (BoxTypePtr drawn_area)
 {
-  render_priv *priv = gport->render_priv;
   int i, ngroups;
   /* This is the list of layer groups we will draw.  */
   int do_group[MAX_LAYER];
